### PR TITLE
1️⃣ refactor(services: rate-limit): migrate from fastify to @whatwg-node/server

### PR DESCRIPTION
**Pull request by YassinEldeeb**
instead of #858 which got closed by accident

### CODE REVIEW

File: packages/services/rate-limit/src/router.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    connectionString: createConnectionString(env.postgres),
+  },
+});
+
+const respondWithTRPC = createFetchAPIHandler({
+  router: rateLimitApiRouter,
+  createContext: async (): Promise<any> => ctx,
+});
+
+rateLimitRouter.all('/trpc/:path+', async (req: any) => {
```
Comment (ardatan) [inline packages/services/rate-limit/src/router.ts:30]:
```suggestion
rateLimitRouter.all('/trpc/:path+', responseWithTRPC)
```
We can pass this handler without an additional `async/await`, no?

**Review by n1ru4l — APPROVED** [review]

**Review by ardatan — APPROVED** [review]

**Comment by n1ru4l** [comment]
I pushed this to https://github.com/kamilkisiela/graphql-hive/pull/1434 and integration tests fail.

**Review by n1ru4l — CHANGES_REQUESTED** [review]
Integration tests fail see https://github.com/kamilkisiela/graphql-hive/pull/1434

**Comment by dotansimha** [comment]
Closing. This PR is outdated and we'll reconsider it later.

**Pull request closed without merging**